src/DirectWebServices/Sectors/EquitySectorsBreakdownConverter.ts

```diff
@@ -0,0 +1,224 @@
+/* *
+ *
+ *  (c) 2009-2025 Highsoft AS
+ *
+ *  License: www.highcharts.com/license
+ *
+ *  !!!!!!! SOURCE GETS TRANSPILED BY TYPESCRIPT. EDIT TS FILE ONLY. !!!!!!!
+ *
+ *  Authors:
+ *  - Pawel Dalek
+ *
+ * */
+
+'use strict';
+
+/* *
+ *
+ *  Imports
+ *
+ * */
+
+import EquitySectorsBreakdownJSON from './EquitySectorsBreakdownJSON';
+import {
+    EquitySectorsBreakdownConverterOptions,
+    EquitySectorsBreakdownMetadata
+} from './EquitySectorsBreakdownOptions';
+import SectorsBreakdown from './SectorsBreakdownOptions';
+import MorningstarConverter from '../../Shared/MorningstarConverter';
+import * as External from '../../Shared/External';
+
+/* *
+ *
+ *  Class
+ *
+ * */
+
+export class EquitySectorsBreakdownConverter extends MorningstarConverter {
+
+    /* *
+     *
+     *  Constructor
+     *
+     * */
+
+    public constructor (
+        options?: EquitySectorsBreakdownConverterOptions
+    ) {
+        super(options);
+
+        // Create main data tables
+        this.tables = [
+            new External.DataTable({ id: 'EqSuperSector' }),
+            new External.DataTable({ id: 'EqSector' }),
+            new External.DataTable({ id: 'EqIndustry' })
+        ];
+
+        this.metadata = {
+            columns: {}
+        };
+    }
+
+    /* *
+     *
+     *  Properties
+     *
+     */
+
+    public readonly metadata: EquitySectorsBreakdownMetadata;
+
+    /* *
+     *
+     *  Functions
+     *
+     * */
+
+    public override parse (
+        options: EquitySectorsBreakdownConverterOptions
+    ): void {
+        const tables = this.tables,
+            metadata = this.metadata,
+            userOptions = { ...this.options, ...options },
+            json = userOptions.json,
+            id = json.identifiers.performanceId,
+            sectorsData = json.morningstarEquitySectorsBreakdown;
+
+        if (sectorsData) {
+            const setSectors = (
+                sectors: string[],
+                table: External.DataTable
+            ) => {
+                // Set sector or super sector column
+                table.setColumn(`Type_${id}`);
```

Comment:
I believe `setColumn` shouldn't be needed here as `setCell` should create columns if they don't exist. Could you please check? 🤔